RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 
 21.3
 -----
+* [***] Adds a smooth, opt-in transition to the Jetpack app. [#19714]
```

Comment:
Looking into this. The `release/21.3` merge caused it to reappear, I think.

Reply:
This was resolved by f1b3975603cc10ab56c88b18a19bb7b915d82c04 which removed the re-addition of this line on the merge commit. I force-pushed it, so heads up @staskus and @salimbraksa. Thanks! 🙇